src/Uno.UI/UI/Xaml/Application.skia.cs

```diff
@@ -129,6 +133,18 @@ private void InvokeOnLaunched()
 		}
 
 		internal void ForceSetRequestedTheme(ApplicationTheme theme) => _requestedTheme = theme;
+
+		private static void PreloadFonts()
+		{
+			if (OperatingSystem.IsBrowser())
+			{
+				// WASM does the font preloading before removing the splash via PrefetchFonts.
+				return;
+			}
+
+			var _ = FontFamilyHelper.PreloadAsync(new FontFamily(FeatureConfiguration.Font.SymbolsFont), FontWeights.Normal, FontStretch.Normal, FontStyle.Normal);
```

Comment:
```suggestion
			_ = FontFamilyHelper.PreloadAsync(new FontFamily(FeatureConfiguration.Font.SymbolsFont), FontWeights.Normal, FontStretch.Normal, FontStyle.Normal);
```

NOTE: That's a fire-and-forget operation. Usually a thing to avoid as it may lead to unobserved exceptions